eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -12,7 +19,7 @@
   <Target Name="UpdateRuntimePack"
           AfterTargets="ResolveFrameworkReferences">
     <ItemGroup>
-      <ResolvedRuntimePack Update="@(ResolvedRuntimePack)" PackageDirectory="$(RuntimePackDir)" />
+      <ResolvedRuntimePack Update="@(ResolvedRuntimePack)" PackageDirectory="$(MicrosoftNetCoreAppRuntimePackRidDir)../../" />
```

Comment:
If you need the parent parent, why don't you use https://github.com/dotnet/runtime/blob/a9c5eadd951dcba73167f72cc624eb790573663a/src/libraries/Directory.Build.props#L153 instead?

Reply:
Because the one that needs to be defined is this one as it is the one used by the Wasm app bundle targets, so this way I only have to pass down one property instead of two.